modules/hooks/activeEffects.js

```diff
@@ -43,21 +43,20 @@ export default function () {
 
         if (effect.parent?.documentName == "Actor" && effect.application == "apply")
         {
-            effect.updateSource({"flags.wfrp4e.effectApplication" : "actor"})
+            await effect.updateSource({"flags.wfrp4e.effectApplication" : "actor"})
         }
 
         if (effect.parent?.documentName == "Actor" && effect.trigger == "oneTime")
         {
             ui.notifications.notify(`${game.i18n.format("EFFECT.Applying", { name: effect.label })}`)
-            game.wfrp4e.utility.applyOneTimeEffect(effect, effect.parent);
+            await game.wfrp4e.utility.applyOneTimeEffect(effect, effect.parent);
             return false
         }
-                
     })
 
 }
 
-function _runUpdateEffects(effect, context, options, id)
+async function _runUpdateEffects(effect, context, options, id)
```

Comment:
This doesn't seem necessary

Reply:
wanted to explicilty show, that those effects are awaitable, i can remove